swupd/bundleinfo.go

```diff
@@ -94,7 +94,12 @@ func (m *Manifest) addFilesFromBundleInfo(c config, version uint32) error {
 	for fpath := range m.BundleInfo.Files {
 		fullPath := filepath.Join(chrootDir, fpath)
 		fi, err := os.Lstat(fullPath)
+		if os.IsNotExist(err) {
+			fmt.Fprintf(os.Stderr, "Warning: Missing file, assuming %%ghost: %s\n", fpath)
+			continue
+		}
 		if err != nil {
+			fmt.Println(err, err.Error())
```

Comment:
This is not the proper way of passing the error to the user. This message error is not showing currently because of a bug on initBundles(), which is the caller of addFilesFromBundleInfo(). 
Patch #417 addresses this issue.

Reply:
Also, is there any other possible error tho Lstat other than IsNotExist? I wonder if this block will ever be executed. (I did a quick search but I couldn't find what are the possible PathErrors that can be returned by Lstat)

Reply:
We should really pass an error here, I'm open to improvements.

And yes, os.Lstat can return lots of errors, like ENOPERM, even ENOMEM (stat allocates a statbuf internally).

Reply:
Ah, indeed. Still, the Print should not be here or it will be printed twice with the fix to initBundles